lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -25,58 +28,205 @@
 NIMROD_DEFAULT = -32767.0
```

Comment:
The MERIDIAN_SCALING_BNG on line 26 is no longer used anywhere in this file, although it could be used on lines 391 and 393

Reply:
Removed